build-system/test-configs/config.js

```diff
@@ -384,6 +384,7 @@ const htmlFixtureGlobs = [
   '!examples/xhr-intercept.html',
   '!test/fixtures/e2e/amp-accordion/amp-accordion.html',
   '!test/fixtures/e2e/amp-accordion/single-expand.html',
+  '!test/fixtures/e2e/amp-analytics/browser-events.html',
```

Comment:
@rsimha This PR is currently blocked on approval from `wg-approvers`, which is a quite authoritative group. 

What do you think about moving this glob set to a different file, so that we can set a more permissive list of OWNERS?

Reply:
This change can currently be approved by anyone on infra or performance. Totally fine with adding other wgs to the list of owners. 

https://github.com/ampproject/amphtml/blob/c491d6f3e767a6749b4e2c003d926111922784b8/build-system/test-configs/OWNERS#L6-L11

Reply:
My idea with extracting the `.html` example list is that modifying it it is lower risk than other configuration objects in this file. I would feel more comfortable allowing a wider set of people to modify that list only. What do you think?

(In the meantime, your approval of this PR would be appreciated since you have powers that I don't. 🙂)

Reply:
Good idea, for which there already is precedent 👍

Once the html list is moved to its own file, we can give everyone permissions to edit it, similar to this file:

https://github.com/ampproject/amphtml/blob/c491d6f3e767a6749b4e2c003d926111922784b8/build-system/test-configs/OWNERS#L12-L15